### PR TITLE
[DDW-1010] Add the usb-detection native module to all 3 installers

### PR DESCRIPTION
**Pull request by michalrus**
Internal PR to add https://github.com/MadLittleMods/node-usb-detection on all three OSes.

@renanvalentin, please, set target branch to yours, and merge it into your branch, after verifying.

/cc @danielmain

### CODE REVIEW

File: source/main/ipc/index.ts [hunk 1]
```diff
@@ -21,8 +21,10 @@ import { handleAddressIntrospectionRequests } from './introspect-address';
 import { handleManageAppUpdateRequests } from './manageAppUpdateChannel';
 import { openExternalUrlChannel } from './open-external-url';
 import { openLocalDirectoryChannel } from './open-local-directory';
+import usbDetect from 'usb-detection';
 
 export default (window: BrowserWindow) => {
+  usbDetect.startMonitoring();
```
Comment (michalrus) [inline source/main/ipc/index.ts:27]: This definitely needs removing – I just wanted to trigger the loading of that native library

**Review by renanvalentin — APPROVED** [review]
Great job @michalrus !

**Comment by renanvalentin** [comment]
@michalrus merged your changes on https://github.com/input-output-hk/daedalus/pull/2900 
Thanks for the amazing job!

**Comment by michalrus** [comment]
Thank _you_! :relieved:

**Pull request closed without merging**